File: Flutter/lib/common/ui_component/audio_effect/store/audio_effect_state.dart
import 'package:flutter/material.dart';

class AudioEffectState {
  ValueNotifier<int> voiceVolume = ValueNotifier(100);
  ValueNotifier<int> musicVolume = ValueNotifier(100);
  ValueNotifier<int> earMonitorVolume = ValueNotifier(100);
  ValueNotifier<bool> enableVoiceEarMonitor = ValueNotifier(false);
  ValueNotifier<int> changerType = ValueNotifier(0);
  ValueNotifier<int> reverbType = ValueNotifier(0);
}

File: Android/tuilivekit/component/audioeffect/src/main/java/com/trtc/uikit/component/audioeffect/store/AudioEffectState.java
package com.trtc.uikit.component.audioeffect.store;

import static com.tencent.liteav.audio.TXAudioEffectManager.TXVoiceChangerType.TXLiveVoiceChangerType_0;
import static com.tencent.liteav.audio.TXAudioEffectManager.TXVoiceReverbType.TXLiveVoiceReverbType_0;

import com.tencent.liteav.audio.TXAudioEffectManager;
import com.trtc.tuikit.common.livedata.LiveData;

public class AudioEffectState {
    public LiveData<Integer>                                 voiceVolume           = new LiveData<>(100);
    public LiveData<Integer>                                 musicVolume           = new LiveData<>(60);
    public LiveData<Integer>                                 earMonitorVolume      = new LiveData<>(100);
    public LiveData<Boolean>                                 enableVoiceEarMonitor = new LiveData<>(false);
    public LiveData<TXAudioEffectManager.TXVoiceChangerType> changerType           =
            new LiveData<>(TXLiveVoiceChangerType_0);
    public LiveData<TXAudioEffectManager.TXVoiceReverbType>  reverbType            =
            new LiveData<>(TXLiveVoiceReverbType_0);

}
